cli/resolver.rs

```diff
@@ -359,24 +435,20 @@ impl MappedSpecifierResolver {
 /// import map, JSX settings.
 #[derive(Debug)]
 pub struct CliGraphResolver {
-  fs: Arc<dyn FileSystem>,
   sloppy_imports_resolver: Option<SloppyImportsResolver>,
   mapped_specifier_resolver: MappedSpecifierResolver,
   maybe_default_jsx_import_source: Option<String>,
   maybe_jsx_import_source_module: Option<String>,
   maybe_vendor_specifier: Option<ModuleSpecifier>,
-  cjs_resolutions: Option<Arc<CjsResolutionStore>>,
-  node_resolver: Option<Arc<NodeResolver>>,
+  node_resolver: Option<Arc<CliNodeResolver>>,
```

Comment:
**question:** It seems odd that `CliNodeResolver` is inside an Arc while each of its fields is also an `Arc`. One or the other should be enough I believe, though I am not quite sure. @mmastrac might probably have a clearer view.

Reply:
Mostly all of these "service" structs are in an Arc. I think it's more convenient because then you don't need to go around refactoring a bunch of code when the internals change and it's a simple pattern to just do and not worry about. Also, I guess in this case it's a question of whether it's faster to clone 4 Arcs when cloning a CliNodeResolver during initialization vs adding more indirection when using a CliNodeResolver. Probably this is accessed way more than cloned, but my guess would be that it's a micro-optimization that doesn't matter, but maybe not.